src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -346,6 +353,13 @@ protected NodeModel CreateAndConnectInputNode(int outputPortIndex, int inputPort
             CommandExecutive.ExecuteCommand(command, UniqueId, ExtensionName);
 
             var inputNode = WorkspaceModel.Nodes.FirstOrDefault(node => node.GUID == command.ModelGuid) as DoubleSlider;
+
+            if (inputNode != null)
+            {
+                // Assign the input slider to the default value of the node's input port
+                var doubleNode = Node.InPorts[inputPortIndex].DefaultValue as DoubleNode;
+                if (doubleNode != null) inputNode.Value = doubleNode.Value;
+            }
```

Comment:
When we create an input slider node, we need to ensure that its current value is the same as the default value of the respective input port of the existing node so that the gizmo stays at its current position and does not jump to any arbitrary default position that's determined by the new slider.